cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -15,23 +15,28 @@ OPERATOR_GIT_HASH=8d3c30de8ec5a9a0c9eeb84ea0aa16ba2395cd68  # release-4.4
 # This function gets a command string and invoke it
 # until the command returns an empty string or until timeout
 function retry {
+  set +x
   local -r tries=$1
   local -r wait_time=$2
   local -r action=$3
   local -r wait_message=$4
 
+  local return_value=${return_value:-1}
```

Comment:
trap "echo finished" RETURN
instead this logic, in order to play with the set,
not that i think we should

Reply:
You mean adding `trap set -x RETURN`, and remove the `set -x &&..` where there is `return`?

Reply:
I like this idea, I will add it to all function that use retry